festival.py

```diff
@@ -23,41 +23,48 @@ def info():
 
     _festival.info()
 
-def textToWavFile(text):
+def textToWavFile(text, filename=None):
     """Returns a filename for a wav file created
     with the text 'text'
-    
+
     Use this method if you are concerned about removing
     the wav file once you have used it.
     Being given the filename, you are responsible for opening it
     as you see fit, and removing it once done
     """
     if not isinstance(text, TEXT_TYPE):
         text = text.decode('utf-8')
-    
-    return _festival._textToWav(text)
 
-def textToWav(text):
+    if filename:
+        file = _festival._textToWav(text, filename)
+    else:
+        file = _festival._textToWav(text)
+    return file
+
+
+def textToWav(text, filename=None):
     """Returns a file object for a wav file created with
```

Comment:
Can you update the docstring to document the use of the `filename` optional argument (and above in `textToWavFile`)